asaps/models.py

```diff
@@ -32,16 +32,13 @@ def search(self, string, repo_id, rec_type):
                     f'"&page_size=100&type[]={rec_type}')
         return self.client.get_paged(endpoint)
 
-    def post_record(self, rec_obj, csv_row, csv_data):
+    def save_record(self, rec_obj):
         """Update ArchivesSpace record with POST of JSON data."""
-        payload = json.dumps(rec_obj)
-        post = self.client.post(rec_obj['uri'], data=payload)
-        logger.info(post.status_code)
-        post = post.json()
-        csv_row['post'] = post
-        csv_data.append(csv_row)
-        logger.info(csv_row)
-        return csv_row
+        data = json.dumps(rec_obj)
+        response = self.client.post(rec_obj['uri'], data=data)
```

Comment:
The asnake client just proxies the HTTP methods to requests, and with requests you can pass the dictionary directly to the `post` method using the `json` argument:

```python
response = self.client.post(rec_obj['uri'], json=rec_obj)
```

This has the added advantage of also setting the `Content-Type` header to `application/json` which is usually what you want.

Reply:
I can't believe I didn't know about that requests argument, thanks! Just fixed it